dev/release/utils-update-docs-versions.py

```diff
@@ -87,3 +88,15 @@
 with open(r_versions_path, 'w') as json_file:
     json.dump(new_r_versions, json_file, indent=4)
     json_file.write("\n")
+
+# Load the updated versions JSON file
+with open(r_versions_path) as json_file:
+    data = json.load(json_file)
+
+# Write HTML to file
+with open(r_html_path, 'w') as html_file:
+    html_file.write('<!DOCTYPE html>\n<html>\n<body>')
+    for i in data:
+        html_file.write('<p><a href="../' + i['version'] + 'r/">' + i['name'] + '</a></p>\n')
```

Comment:
I think that `i['version']` and `i['name']` will not include any HTML special characters such as `<` but how about escape them for safety?

```python
import html
html.escape(i['version'])
```

How about using f-string?

```suggestion
        html_file.write(f'<p><a href="../{i["version"]}/r/">{i["name"]}</a></p>\n')
```

Reply:
Thanks, done!